### ISSUE TITLE
Species native in one Belgian region, but not in another

### ISSUE THREAD

**Issue by timadriaens** [trias-project/unified-checklist#49]
A problem (already raised in #32) arising from including regional distributions in the unified checklist (see #45 and #43) is that eventually, species that have been introduced in one region but are native in another, are included in the checklist of alien species of Belgium. I think it would be best to not include those in the unified? Some examples are 

- Podarcis muralis is there because it is non-native to Flanders (introduced with habitat material probably and still spreading along the railway network). 
- Natrix helvetica which was introduced in Flanders and Brussels but is native in Wallonia (and of conservation concern everywhere)

@damianooldoni @peterdesmet We should decide what to do with such cases, because it is strange they appear on a Belgian alien species checklist.

**Comment by damianooldoni** [comment]
I understand your point, @timadriaens. About the examples you mention:
- _Podarcis muralis_ comes from RINSE: https://www.gbif.org/species/145954456 and it has distribution location Belgium only (see distributions:http://api.gbif.org/v1/species/145954456/distributions)
- _Natrix helvetica_ comes from Ad hoc checklist: https://www.gbif.org/species/148437871 and it has distribution location `Flemish region` only

I think that the unified checklist, which is published as the [_Global Register of Introduced and Invasive Species - Belgium_](https://www.gbif.org/dataset/6d9e952f-948c-4483-9807-575348147c7e), should contain **only** taxa which are alien in **all** Belgium. As written in checklist's general description on GBIF:

> It contains information on 2,500+ validated non-native taxa in **Belgium**.

As Belgium = Flanders + Brussels + Wallonia, I read the sentence above as follows:
non-native in **Belgium** = non-native in **Flanders** AND non-native in **Brussels** AND non-native in **Wallonia**

Or logically speaking with symbols: the function _non_nativity_ is a AND function with respect to sum:
```r
non_nativity(a+b) = non_nativity(a) AND non_nativity(b)
```

The other interpretation, where the function _non_nativity_ is a OR function with respect to sum:
```r
non_nativity(a+b) = non_nativity(a) OR non_nativity(b)
```
sounds me quite strange and can arise cases as mentioned by @timadriaens above. The problem is that sometimes the error is at checklist level (see example 1) so we start with wrong information.

**Comment by timadriaens** [comment]
I agree that for the unified checklist of Belgium we only have to consider species that are non-native in all regions (so the and-function). Some species are non-native to Belgium and have only been recorded in one region, sometimes the only thing we know is that it's non-native to Belgium and we don't have regional information. On the other hand, if it is present in one region, it is also present in Belgium of course.

scientific name | ... | location | country code | occurrence status | date first observation | date last observation | degree of establishment
-- | -- | -- | -- | -- | -- | -- | --
scientific name | ... | Flanders | BE | present | 2008 | 2012 | reproducing
scientific name | ... | Wallonia | BE | present | 2012 |   | reproducing
scientific name | ... | Wallonia | BE | present | 2012 |   | reproducing

In fact the RINSE checklist is correct because the working area of that project was entirely in Flanders in that case. And the add hoc for Natrix is also correct as the species IS non-native but only in Flanders... So in order for the AND function to work properly the regional distribution should be filled for all the species? Which means we have to triplicate every record?

**Comment by damianooldoni** [comment]
As the problem of these two species comes back while discussing indicators, this problem, already theoretically important, becomes in practice also relevant, and therefore should be solved.
The solution, I think, is adding the DwC term `establishmentMeans` with value `native` for Wallonia in the checklist distribution. @timadriaens , @peterdesmet , @qgroom , what do you think about it?

**Comment by timadriaens** [comment]
yes, that should do it I think.
